centreon/www/api/class/centreon_ceip.class.php

```diff
@@ -158,7 +158,11 @@ private function getVisitorInformation(): array
             } else {
                 $role = 'User';
             }
-        } else { 
+
+            if (array_key_exists('IS_TRIAL', $_ENV) && $_ENV['IS_TRIAL']) {
```

Comment:
This condition is an isset() :
```suggestion
            if (isset($_ENV['IS_TRIAL'])) {
```

Reply:
Ticket was not up to date with recents changes, this variable no longer exists I'll update the PR

Reply:
Ok I resolve and approve this PR ;) Thanks